scripts/docs-collator/templates/modules/terraform-docs.yml

```diff
@@ -187,6 +123,62 @@ content: |-
   </dl>
   {{- end }}
 
+  ## Dependencies
+
+  {{/* TODO This is temporary, hard-coded value. We need to write a function to pull the terraform version since it isnt natively supported by terraform-docs */}}
```

Comment:
It's definitely supported by terraform docs. 

Example: https://github.com/cloudposse/terraform-aws-components/tree/main/modules/acm#requirement_terraform